operator/pkg/helmreconciler/reconciler.go

```diff
@@ -94,17 +94,19 @@ func NewHelmReconciler(client client.Client, restConfig *rest.Config, iop *value
 	if opts.ProgressLog == nil {
 		opts.ProgressLog = progress.NewLog()
 	}
-	if waitForResourcesTimeoutStr, found := os.LookupEnv("WAIT_FOR_RESOURCES_TIMEOUT"); found {
-		if waitForResourcesTimeout, err := time.ParseDuration(waitForResourcesTimeoutStr); err == nil {
-			opts.WaitTimeout = waitForResourcesTimeout
+	if int64(opts.WaitTimeout) == 0 {
```

Comment:
I don't understand this condition, why it needs to check for `WaitTimeout==0`

Reply:
If the `WaitTimeout` is not zero value, then apply the default value.